indico/modules/designer/client/js/index.js

```diff
@@ -240,11 +244,28 @@ import {$T} from 'indico/utils/i18n';
     var itemStyles = _.filter([item.bold ? 'bold' : null, item.italic ? 'italic' : null]);
 
     // Change the selectors so that they match the properties of the item
+    $('#text-color-selector').val(item.color);
+    var textColorPreview = $('#text-color-preview');
+    if (textColorPreview.hasClass('no-value') && item.color.length > 1) {
+      textColorPreview.removeClass('no-value');
+    }
+    textColorPreview.attr('style', 'background-color: ' + item.color + '; opacity: 1;');
```

Comment:
You can use `.css({...})` to set the style directly instead of using a css string